k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1497,7 +1497,8 @@ public void onMessageBuildSuccess(MimeMessage message, boolean isDraft) {
                 message.setUid(mMessageReference.getUid());
             }
 
-            boolean saveRemotely = recipientPresenter.isAllowSavingDraftRemotely();
+            // TODO more appropriate logic here? not sure
+            boolean saveRemotely = !recipientPresenter.getCurrentCryptoStatus().shouldUsePgpMessageBuilder();
```

Comment:
I don't think it should live in the Activity either which is probably the issue here. It should really be in a controller, but let's leave that for another PR.